liblwgeom/liblwgeom.h.in

```diff
@@ -1833,7 +1833,7 @@ extern size_t gserialized_from_lwgeom_size(const LWGEOM *geom);
 * will contain the size of the final output, which is useful for setting the PgSQL 
 * VARSIZE information.
 */
-extern GSERIALIZED* gserialized_from_lwgeom(LWGEOM *geom, int is_geodetic, size_t *size);
+extern GSERIALIZED* gserialized_from_lwgeom(LWGEOM *geom, size_t *size);
```

Comment:
@pramsey   Can you look at this before my pull becomes stale and I have to redo it again.